sdk/python/tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = py27, py3
```

Comment:
No need to support Python 2z

Reply:
sure. I fixed by hand for this time. But it will be better if I can figure out how to configure openapi-generator-cli to omit `py27` in `envlist`